### PR TITLE
MO Transformation configs are missing in openvino-dev package

### PR DESCRIPTION
**Pull request by slyubimt**
### Details:
 - *Add MO Transformation configs*

### Tickets:
 - *53581*

### CODE REVIEW

File: model-optimizer/setup.py [hunk 1]
```diff
@@ -9,23 +9,22 @@
...
 
-package_name = 'mo'
+PACKAGE_NAME = 'mo'
 
 # Detect all the framework specific requirements_*.txt files.
 requirements_txt = []
 py_modules = []
 for name in os.listdir():
-    if re.match('requirements(.*)\.txt', name):
+    if re.match(r'requirements(.*).txt', name):
```
Comment (IRDonch) [inline model-optimizer/setup.py:25]:
```suggestion
    if re.match(r'requirements(.*)\.txt', name):
```

File: model-optimizer/setup.py [hunk 2]
```diff
@@ -9,23 +9,22 @@
...
         requirements_txt.append(name)
-    if re.match('mo(.*)\.py', name):
+    if re.match(r'mo(.*).py', name):
```
Comment (IRDonch) [inline model-optimizer/setup.py:27]:
```suggestion
    if re.match(r'mo(.*)\.py', name):
```

**Review by IRDonch — CHANGES_REQUESTED** [review]

**Review by IRDonch — APPROVED** [review]

**Comment by lazarevevgeny** [comment]
@slyubimt , @dkurt , some time ago I asked to compare the MO packaged into a wheel and the MO which goes into the PSET installer. Now it looks like some files are still missing. Can we add a test which will check that invariant after each MO commit?

File: model-optimizer/setup.py [hunk 3]
```diff
@@ -92,6 +91,11 @@ def find_package_modules(self, package, package_dir):
               'mo = mo.__main__:main',
            ],
       },
+      package_data={
```
Comment (lazarevevgeny) [inline model-optimizer/setup.py:94]: One more file is missing: `mo/front/caffe/proto/mo_caffe.proto`.

Please, add a test to compare content of wheel package and files built with CMake.

**Review by lazarevevgeny — CHANGES_REQUESTED** [review]

File: model-optimizer/setup.py (hunk 3 above)
Comment (lazarevevgeny) [reply model-optimizer/setup.py:94]: May be more files are missing, but this one which I was able to find quickly.

Comment (dkurt) [reply model-optimizer/setup.py:94]: `mo_caffe.proto` is not used by MO because there are generated `caffe_pb2.py` files, isn't it?

Comment (lazarevevgeny) [reply model-optimizer/setup.py:94]: It is not used, but we still deliver this file as part of regular installer to allow users re-generate the `caffe_pb2.py` themselves. So I would request to add this file to a pip distribution as well.

**Review by lazarevevgeny — APPROVED** [review]

**Comment by lazarevevgeny** [comment]
> @slyubimt , @dkurt , some time ago I asked to compare the MO packaged into a wheel and the MO which goes into the PSET installer. Now it looks like some files are still missing. Can we add a test which will check that invariant after each MO commit?

@slyubimt should I expect the unit test to be added as part of this PR?

**Comment by slyubimt** [comment]
@lazarevevgeny yes, test configuration scope for wheels will be extended by networks with transformations configs.

**Pull request merged**